commit/merkleroot/rmn/client.go

```diff
@@ -0,0 +1,19 @@
+package rmn
+
+import (
+	"context"
+	"errors"
+)
+
+// ErrTimeout is returned when the signature computation times out.
+var ErrTimeout = errors.New("signature computation timeout")
+
+// Client contains the methods required by the plugin to interact with the RMN nodes.
+type Client interface {
+	// ComputeSignatures computes and returns the signatures for the provided lane updates.
+	ComputeSignatures(
+		ctx context.Context,
+		destChain DestChainInfo,
+		requestedUpdates []FixedDestLaneUpdateRequest,
```

Comment:
Mentioned above, would be great to express uniqueness by using a map here:
```suggestion
		requestedUpdates map[ChainSelector]FixedDestLaneUpdateRequest,
```

Reply:
Replied above